util.py

```diff
@@ -200,7 +200,7 @@ def get_hostname(config=None):
                 return gce_hostname
 
     # Try to get the docker hostname
-    docker_util = DockerUtil()
+    docker_util = DockerUtil(agentConfig=config)
```

Comment:
it's used [in DockerUtil](https://github.com/DataDog/dd-agent/blob/haissam/fine-grained-reload/utils/dockerutil.py#L55-L58) to get the instance of the config_store. We could pass a subset of it instead since we basically just need the `sd_config_backend` field, but that would be more confusing than useful imho.